src/dymaptic.GeoBlazor.Core/Components/Widgets/LayerList/ListItem.cs

```diff
@@ -0,0 +1,29 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Layers;
+using Microsoft.AspNetCore.Components;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace dymaptic.GeoBlazor.Core.Components.Widgets.LayerList;
+
+public class ListItem 
+{
+    [Parameter]
+    public string? Title { get; set; }
+    [Parameter]
+    public Layer? Layer { get; set; }
+    [Parameter]
+    public bool? Visible { get; set; }
+    public List<ListItem>? Children { get; set; }
```

Comment:
I'm a big fan of using whitespace (blank lines) to make things easier to visualize/read. I'm not 100% consistent about it, but in general, I find it better to add space above attributes and xml comments.